app/models/proxy.rb

```diff
@@ -476,7 +478,7 @@ class PolicyConfig
 
     attr_accessor :name, :version, :configuration, :enabled
 
-    delegate :to_json, :as_json, to: :to_h
+    delegate :to_json, :to_yaml, to: :as_json
```

Comment:
It is correct to delegate to `as_json` because afaik it should adhere to certain rules while `to_h` has not so well defined behavior that may not be so apparent to a reader. Is also alright if I switch `to_h` to be the alias?

Reply:
OK, I saw this behavior is standard on Rails. This is OK then.